src/CTA.WebForms2Blazor/ControlConverters/LabelControlConverter.cs

```diff
@@ -9,19 +11,28 @@ public class LabelControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["id"] = "for"
+                };
             } 
         }
-        protected override string BlazorName { get { throw new NotImplementedException(); } }
-        
-        public LabelControlConverter() : base()
-        {
-            
-        }
+        protected override string BlazorName { get { return "label"; } }
+
+        protected override string NodeTemplate { get { return @"{2}"; } }
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            var textAttr = node.Attributes.AttributesWithName("text").Single();
+            var labelText = textAttr.Value;
```

Comment:
Same as above, is it possible for the text attribute to be missing? If so, would calling .Single() throw an error?